erts/emulator/beam/emu/instrs.tab

```diff
@@ -1086,28 +1086,36 @@ catch(Y, Fail) {
 }
 
 catch_end(Y) {
+    /*
+     * At entry:
+     *
+     *    x0 = THE_NON_VALUE
```

Comment:
`x0` can be a value here as well, right?

Reply:
Yes, you are right. I have force-pushed a correction.